server/players.py

```diff
@@ -44,12 +44,12 @@ def __init__(
         self.session = session
 
         self.ratings = PlayerRatings(
-            default=(config.START_RATING_MEAN, config.START_RATING_DEV)
+            lambda: (config.START_RATING_MEAN, config.START_RATING_DEV)
         )
         if ratings is not None:
             self.ratings.update(ratings)
 
-        self.game_count = RatingTypeMap(0)
+        self.game_count = RatingTypeMap(int)
```

Comment:
I don't like this `RatingTypeMap` with default factory. Default value = is fine. but factory - you're not using it at all, only here. Another place is `RatingTypeMap` with `None` and another place is `PlayerRatings` with default rating function (min, max).

Hey, how about to just use "default values" without any other holy magic? :-)

Reply:
Because then I have to put default creation logic in multiple places instead of just one place. Plus adding additional checks for containment everywhere. Why? What is the point of adding all that extra code for no reason?